lib/repl.js

```diff
@@ -149,7 +155,7 @@ function REPLServer(prompt,
   self.useGlobal = !!useGlobal;
   self.ignoreUndefined = !!ignoreUndefined;
   self.replMode = replMode || exports.REPL_MODE_SLOPPY;
-  self.underscoreAssigned = false;
+  self.underscoreAssigned = self.noUnderscore = !!noUnderscore;
```

Comment:
I don't see where `self.noUnderscore` is ever assigned a value. I think this will always be `undefined`. Am I missing something?